jsdoc.json

```diff
@@ -0,0 +1,23 @@
+{
+    "tags": {
+        "allowUnknownTags": true,
+        "dictionaries": ["jsdoc"]
+    },
+    "source": {
+        "include": ["addon/"],
+        "includePattern": ".+\\.js(doc)?$",
+        "excludePattern": "(^|\\/|\\\\)_"
+    },
+    "plugins": ["plugins/markdown"],
+    "templates": {
+        "cleverLinks": false,
+        "monospaceLinks": false
+    },
+    "opts": {
+        "access": "all",
+        "destination": "./docs",
+        "encoding": "utf8",
+        "readme": "README.md",
+        "recurse": true
+    }
+}
```

Comment:
Do we want this file included when it does not contain all of the plugins we would use as well as there is currently not a workable, one-step-installable JSDoc release yet?  Seems this is might be "ahead of its time".